packages/ref-doc/src/extractor.ts

```diff
@@ -143,6 +145,15 @@ function extractOperationRefDoc(program: Program, operation: Operation): Operati
 function extractDecoratorRefDoc(program: Program, decorator: Decorator): DecoratorRefDoc {
   const paramDoc = getParmeterDocs(decorator);
   const parameters: FunctionParameterRefDoc[] = decorator.parameters.map((x) => {
+    const docVal = paramDoc.get(x.name);
+    if (docVal === undefined || docVal === "") {
+      reportDiagnostic(program, {
```

Comment:
despite using those diagnositics I would maybe not add them to the program like that, maybe just do what you add before but with `createDiagnostic`